etc/service-bus.api.ts

```diff
@@ -0,0 +1,411 @@
+}
+
+// @public
+export function delay<T>(t: number, value?: T): Promise<T>;
+
+// @public (undocumented)
+interface Delivery {
+  // (undocumented)
+  accept(): void;
+  // (undocumented)
+  readonly format: number;
+  // (undocumented)
+  readonly id: number;
+  // (undocumented)
+  readonly link: Sender | Receiver;
+  // (undocumented)
+  modified(params?: ReleaseParameters): void;
+  // (undocumented)
+  reject(error?: AmqpError): void;
+  // (undocumented)
+  release(params?: ReleaseParameters): void;
+  // (undocumented)
+  readonly remote_settled: boolean;
+  // (undocumented)
+  readonly remote_state?: DeliveryOutcome;
+  // (undocumented)
+  readonly sent: boolean;
+  // (undocumented)
+  readonly settled: boolean;
+  // (undocumented)
+  readonly state?: DeliveryOutcome;
+  // (undocumented)
+  readonly tag: Buffer | string;
+  // (undocumented)
+  update(settled: boolean, state?: any): void;
+}
+
+// @public
+interface Dictionary<T> {
+  // (undocumented)
+  [key: string]: T;
+}
+
+// @public (undocumented)
+interface MessageHandlerOptions {
+  autoComplete?: boolean;
+  maxAutoRenewDurationInSeconds?: number;
+  maxConcurrentCalls?: number;
+}
+
+// @public
+interface MessageHeader {
+  delivery_count?: number;
+  durable?: boolean;
+  first_acquirer?: boolean;
+  priority?: number;
+  ttl?: number;
+}
+
+// @public
+interface MessageProperties {
+  absolute_expiry_time?: number;
+  content_encoding?: string;
+  content_type?: string;
+  correlation_id?: string | number | Buffer;
+  creation_time?: number;
+  group_id?: string;
+  group_sequence?: number;
+  message_id?: string | number | Buffer;
+  reply_to?: string;
+  reply_to_group_id?: string;
+  subject?: string;
+  to?: string;
+  user_id?: string;
+}
+
+// @public
+class MessageSession extends LinkEntity {
+  // WARNING: The type "ClientEntityContext" needs to be exported by the package (e.g. added to index.ts)
+  // WARNING: The type "MessageSessionOptions" needs to be exported by the package (e.g. added to index.ts)
+  constructor(context: ClientEntityContext, options?: MessageSessionOptions);
+  autoComplete: boolean;
+  autoRenewLock: boolean;
+  close(): Promise<void>;
+  // WARNING: The type "ClientEntityContext" needs to be exported by the package (e.g. added to index.ts)
+  // WARNING: The type "MessageSessionOptions" needs to be exported by the package (e.g. added to index.ts)
+  static create(context: ClientEntityContext, options?: MessageSessionOptions): Promise<MessageSession>;
+  getState(): Promise<void>;
+  isOpen(): boolean;
+  maxAutoRenewDurationInSeconds: number;
+  maxConcurrentCallsPerSession?: number;
+  maxConcurrentSessions?: number;
+  maxMessageWaitTimeoutInSeconds: number;
+  peek(messageCount?: number): Promise<ReceivedMessageInfo[]>;
+  peekBySequenceNumber(fromSequenceNumber: Long, messageCount?: number): Promise<ReceivedMessageInfo[]>;
+  // WARNING: The type "SessionHandlerOptions" needs to be exported by the package (e.g. added to index.ts)
```

Comment:
This has already been exported in a separate PR. Pulling from master branch should get rid of this.